src/editors/data/redux/thunkActions/problem.js

```diff
@@ -1,10 +1,13 @@
 import _ from 'lodash-es';
 import { actions } from '..';
+import * as requests from './requests';
 import { OLXParser } from '../../../containers/ProblemEditor/data/OLXParser';
 import { parseSettings } from '../../../containers/ProblemEditor/data/SettingsParser';
 import { ProblemTypeKeys } from '../../constants/problem';
 import ReactStateOLXParser from '../../../containers/ProblemEditor/data/ReactStateOLXParser';
 import { blankProblemOLX } from '../../../containers/ProblemEditor/data/mockData/olxTestData';
+import { camelizeKeys } from '../../../utils';
+import * as module from './problem';
```

Comment:
I generally don't think self-imports are a good idea - they make the application logic more convoluted and add unnecessary complexity, and are not normally used in React applications. Is this really necessary? Why do we need to mock out `loadProblem`, and if we do, why do we not have it in a separate module?